style.css

```diff
@@ -0,0 +1,195 @@
+:root{
+    --main-font: 'Roboto', Arial, sans-serif;
+    --clr-primary: rgb(3,3,3);
+    --bgr-accent: rgb(204,0,0);
+    --clr-accent: rgb(255,255,255);
+    --clr-secondary: rgb(96,96,96);
+    --clr-button-border: rgb(238,238,238);
+}
+
+*,
+*::after,
+*::before {
+    box-sizing: border-box;
+}
+
+html {
+    box-sizing: border-box;
+    font-family: var(--main-font);
+    color: var(--clr-primary);
+    font-size: .8rem;
+}
+
+iframe {
+    width: 100%;
+    min-height: 70vh;
+}
+
+img {
+    max-width: 100%;
+    max-height: 100%;;
+}
+
+header {
+    height: 56px;
+    display: flex;
+    padding: 0;
+    align-items: center;
+    justify-content: space-between;
+    flex-direction: row;
+    width: 100%;
+}
+
+header > * {
+    vertical-align: middle;
+    flex-grow: 1;
+    flex-shrink: 1;
+    display: flex;
+}
+
+#search-bar {
+    height: 35px;
+    display: flex;
+}
+
+#search-bar > * {
+    height: 100%;
+    margin: 0;
+    padding: 0;
+}
+
+#search input, button{
```

Comment:
If you need this elements to have similar formatting it would be better to create a separate class with the general format and use the individuals classes for the specifics. You may also need to edit the html accordingly.